project-koku/koku
# Use sources UUID on provider UUID when creating new sources

@lcouzens requested that we use the sources UUID for the provider UUID when adding new cost management sources to simplify the test automation.

**Testing**
1.  Add an AWS provider directly to the koku api
2.  Add Source with same values as AWS source from step 1.  Verify that it's promoted to a platform source (koku_uuid != source_uuid)
3.  Add a new Source and verify that the koku_uuid matches the source_uuid.

**Test Results**
[use_source_uid_for_provider_ut.txt](https://github.com/project-koku/koku/files/3871151/use_source_uid_for_provider_ut.txt)
